src/js/components/MarathonTaskDetailsList.js

```diff
@@ -58,6 +58,14 @@ class MarathonTaskDetailsList extends React.Component {
     return task.status;
   }
 
+  getTimeField(time) {
+    return (
+      <time dateTime={time} title={time}>
+        {new Date(time).toLocaleString()}
```

Comment:
@leemunroe sure thing. 

![task detail panel](https://s3.amazonaws.com/f.cl.ly/items/320J0Y2b2H1F3o360X1X/Image%202016-05-30%20at%2011.15.12%20AM.png?v=fb9327c2)

This is about how the `Staged At`/`Started At` values should be formatted. The data from the server is an [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) combined date and time (like the `Version` value), which isn't so readable. 

The method which Marathon uses and which Felix is using here
* Formats the date according to the user's configuration
* Converts the date to the user's local timezone

I wondered whether this is still our preferred way of doing things. 

NB this discussion shouldn't block merge on this PR.